README.md

```diff
@@ -43,6 +43,11 @@ Regardless of how you want to run FediFetcher, you must first get an access toke
    3. Save
    4. Copy the value of `Your access token`
 
+> **Note**
```

Comment:
Thank you for your PR! 

can you explain how this is different from Mastodon? It's the same, isn't it?

Reply:
With Mastodon (and also Sharkey, as it happens) you can use the auth token generated in the UI. With Iceshrimp, you specifically have to go through the OAuth process as described.

Reply:
Ah, OK, that makes sense, thanks.

In that case can we refer to the OAuth instructions that are found within the very next paragraph please, rather than to the Mastodon docs: https://github.com/nanos/FediFetcher?tab=readme-ov-file#if-you-are-not-an-admin-on-your-instance (unless, of course, these don't work/apply with Iceshrimp).

Reply:
Those should apply to Iceshrimp as well. The difference here is that Sharkey and Firefish (AFAIK) use the same tokens to access both the Misskey API and the Mastodon API (so a token generated for one will work with the other) whereas Iceshrimp has separate tokens for it's Misskey and Mastodon APIs.

(p.s. Sharkey isn't a fork of Firefish)